source/jormungandr/tests/journey_common_tests.py

```diff
@@ -443,6 +443,11 @@ def test_sp_to_sp(self):
 
         query = "journeys?from=stop_point:uselessA&to=stop_point:stopB&datetime=20120615T080000"
 
+        if self.data_sets.get('main_routing_test', {}).get('scenario') == 'distributed':
+            # In distributed scenario, we should deactivate direct path so that we can reuse the same
```

Comment:
Is that a `TODO` ?
I don't understand which code could be reused, and where direct_path should be deactivated.
I probably miss context.

Reply:
To be applied on every occurrence if it is accurate:
```suggestion
            # In distributed scenario, we must deactivate direct path so that the behavior is the same as in the new_default scenario (as distributed scenario keeps longs direct_paths by default)
```